wwwapp/models.py

```diff
@@ -90,28 +90,35 @@ class Meta:
     Returns a date class instance or None if the PESEL is malformed or not present.
     """
     def get_birth_date(self) -> date or None:
+        if not self.pesel.isdigit():
+            return None
+        if len(self.pesel) != 11:
+            return None
+        # https://pl.wikipedia.org/wiki/PESEL#Cyfra_kontrolna_i_sprawdzanie_poprawno%C5%9Bci_numeru
```

Comment:
Change to:
https://en.wikipedia.org/wiki/PESEL#Format

Reply:
:man_facepalming: